frontend/components/atoms/AUserName.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Box } from '@material-ui/core';
+import Image from 'next/image';
+
+interface AUserNameProps {
+  username: string;
+  avatar?: string;
+  // hasAvatar: boolean;
+  // isLiked: boolean;
+  // isUsed: boolean;
+  // isListing: boolean;
+  // username: string;
+  // title: string;
+  // avatar?: string
+}
+
+const AUserName: FC<AUserNameProps> = ({ username, avatar, ...other }) => (
```

Comment:
Should AUsername should be shifted to molecule instead? 
Atoms are components that cannot be broken down further, such as a single button or a single label, while this is a component made of an image and a paragraph.